tests/Feature/Service/Student/UpdateStudentImageServiceTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Tests\Unit\Service\Student;
+
+use App\Models\Student;
+use App\Service\Student\UpdateStudentImageService;
+use App\Exceptions\StudentNotFoundException;
+use App\Http\Controllers\api\Student\UpdateStudentImageController;
+use App\Http\Requests\UpdateImageStudentRequest;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Illuminate\Http\UploadedFile;
+use Illuminate\Support\Facades\Storage;
+use Tests\TestCase;
+
+class UpdateStudentImageServiceTest extends TestCase
+{
+    use DatabaseTransactions;
+	
+	protected UpdateStudentImageService $studentUpdateImageService;
+	private string $photos_path = 'public/photos/';
+	
+	
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->studentUpdateImageService = new UpdateStudentImageService();
+    }
+    
+	
+	
+    /** @test */
+    public function it_updates_student_image_path_in_database()
```

Comment:
the same. Do not use the comment `@test` and use the formula "testCanDoSomethingWhenSomethingHappens"